riscv_analysis_cli/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+pub mod wrapper {
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Serialize, Deserialize, Debug)]
+    pub struct ListWrapper {
+        pub diagnostics: Vec<DiagnosticWrapper>,
+    }
+
+    #[derive(Serialize, Deserialize, Debug)]
+    pub struct DiagnosticWrapper {
```

Comment:
If the underlying types `Diagnostic`, `Range`, and `Position` implement `Serialize` and `Deserialize`, we don't need a wrapper type for this. We can simply use the `serde_json` methods on those types.

~On the other hand, if we want to customize the way the values are outputted, then we would want to do this. Is there any major reason for that?~

I see that we are doing this to hardcode the path. I would call these types something else other than `Wrapper`. Maybe `TestCase` or `Hardcoded` or something similar.